Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -71,7 +71,63 @@ class CredentialsManagerSpec: QuickSpec {
             it("should fail to clear credentials") {
                 expect(credentialsManager.clear()).to(beFalse())
             }
+            
+        }
+        
+        describe("multi instances of credentials manager") {
```

Comment:
I'm not as bothered about tests as long as they work. However, they would look cleaner to use closure shorthand like https://github.com/danielphillips/Auth0.swift/blob/a78a4018722c9202ee5d1a75c2608f5c00f855ac/Auth0Tests/CredentialsManagerSpec.swift#L263

```
credentialsManager.credentials { 
   expect($1?.accessToken) == AccessToken
   expect($1?.idToken) == IdToken
  done()
}
```

Cut down on the extra params.

Reply:
Good point thank you. Done that now.